french/dessine/miel/ouvrir-porte.md

```diff
@@ -0,0 +1,6 @@
+Vous ouvrez la porte en miel mais, telle une mouche sur du papier tue-mouche, votre main reste collée à la poignée...
+Vous décidez de vous aider de votre seconde main pour vous libérer, cependant, celle-ci se retrouve aussi engluée dans ce miel.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.